user_templates/mcu_defaults/main/BTT_SKR_E3mini_v3.cfg

```diff
@@ -0,0 +1,30 @@
+
+#----------------------------------------#
+#### BTT SKR Mini E3 V3 definition ########
+#----------------------------------------#
+
+[mcu]
+##--------------------------------------------------------------------
+serial: /dev/serial/by-id/change-me-to-the-correct-mcu-path
+##--------------------------------------------------------------------
+
+[include config/mcu_definitions/BTT_SKR_E3mini_v3.cfg] # Do not remove this line
+[board_pins SKR_mini_e3_mcu]
+mcu: mcu
+aliases:
+    X_STEP=MCU_XM_STEP , X_DIR=MCU_XM_DIR , X_ENABLE=MCU_XM_ENABLE , X_TMCUART=MCU_XM_UART ,
+    Y_STEP=MCU_YM_STEP , Y_DIR=MCU_YM_DIR , Y_ENABLE=MCU_YM_ENABLE , Y_TMCUART=MCU_YM_UART ,
+    Z_STEP=MCU_ZM_STEP , Z_DIR=MCU_ZM_DIR , Z_ENABLE=MCU_ZM_ENABLE , Z_TMCUART=MCU_XM_UART ,
+    E_STEP=MCU_EM_STEP , E_DIR=MCU_EM_DIR , E_ENABLE=MCU_EM_ENABLE , E_TMCUART=MCU_EM_UART ,
+
```

Comment:
You are missing here the UART and TX of the other file i commented.

Reply:
Hi @Surion79 sorry if those are missing I adapted the existing SKR Mini E3 v2 config already in the repo for this board revision.  I'll take a look and revise if needed

Reply:
the v2 is unfortunately not (longer) working. assigning multiple aliases to same pin results in an error. The uart and tx must be implemented by overriding the tmc config by adding it in the overrides.cfg. I'll get a PR for a working version read and update the v2 accordingly. what changes are between v2 and v3?

Reply:
@Surion79 got it - I didn't realize that.  The only changes from V2 to V3 are the changing of some pin assignments (I think on one of the motor pins) and the addition of a FAN2 port that is typically used for controller fans, but can be for anything.  I don't mind updating my config - do you have an example I can follow?

Reply:
resolving conversation per our other discussions